docs/static/governance/avm-standard-github-labels.csv

```diff
@@ -25,6 +25,7 @@ Status: Module Orphaned :eyes:,The module has no owner and is therefore orphaned
 Status: Response Overdue :triangular_flag_on_post:,When an issue/PR has not been responded to for X amount of days,850000
 Status: Looking For Assistance :duck:,This item is looking for anyone to help develop the code and submit a PR for resolution,03FCC2
 Type: Bug :bug:,Something isn't working,D73A4A
+Type: CI :rocket:,This issue is related to the AVM CI,74CFB0
```

Comment:
Not that it matters much, but wouldn't `Class: CI` be more appropriate than  `Type: CI `?

Reply:
@eriqua, in [SNFR23](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr23---category-contributionsupport---github-repo-labels), we only used "class" as the abbreviated version of the "module classification: resource/pattern/utility". Everything else was defined as a "type", hence I went with type. Unless you have something strongly against it, at this point, I would stick to this convention.

Reply:
Fair @matebarabas if that was the agreement no need to change it